test/rules/noRequireImportsTests.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2013 Palantir Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+describe("<no-require-imports>", () => {
+
+    it("forbids CommmonJS style imports", () => {
+        var fileName = "rules/norequireimports.test.ts";
+        var Rule = Lint.Test.getRule("no-require-imports");
+
+        var actualFailures = Lint.Test.applyRuleOnFile(fileName, Rule);
+
+        assert(actualFailures.length === 4, "Expected four failures");
+        var failure1 = Lint.Test.createFailuresOnFile(fileName, Rule.FAILURE_STRING + "'lib'")([1, 5], [1, 25]);
+        var failure2 = Lint.Test.createFailuresOnFile(fileName, Rule.FAILURE_STRING + "'lib2'")([3, 5], [3, 27]);
+        var failure3 = Lint.Test.createFailuresOnFile(fileName, Rule.FAILURE_STRING + "'lib5'")([11, 5], [11, 27]);
+        var failure4 = Lint.Test.createFailuresOnFile(fileName, Rule.FAILURE_STRING + "'lib6'")([12, 5], [12, 27]);
+
+
```

Comment:
extra newline --- this should be picked up by the linter; we should investigate why that's not the case.